package.json

```diff
@@ -109,7 +109,19 @@
                 "command": "extension.ccFindCheckouts",
                 "key": "ctrl+j ctrl+f"
             }
-        ]
+        ],
+        "menus": {
+            "explorer/context": [
```

Comment:
Since both are visible on files and directory I suggest we keep only one command ( called "Clearcase: Update" ) that detect whenever to update a directory or a file ( actually that should be trivial because you just have to pass the path to "cleartool update" command )